README.md

```diff
@@ -1,4 +1,5 @@
 # ansible-role-simple-mysql-backups
+![.github/workflows/molecule.yml](https://github.com/Chithrak07/ansible-role-simple-mysql-backups/workflows/.github/workflows/molecule.yml/badge.svg)
```

Comment:
Also, I wonder if we it's possible to make this clear that this pass/fail badge is for the role setup, and not backer-upper itself. All this role does is install a cron job and upload the backup scripts, which although it could fail, isn't really indicative of what the role's purpose is.  Advertising a pass on just the cron job installation seems like it could be misleading. Let me know your thoughts here.